lms/djangoapps/instructor_analytics/csvs.py

```diff
@@ -31,7 +31,7 @@ def create_csv_response(filename, header, datarows):
         quoting=csv.QUOTE_ALL)
 
     disclaimer = SensitiveMessageOnReports()
-    disclaimer.csv_direct(csvwriter)
+    csvwriter = disclaimer.csv_direct(csvwriter)
```

Comment:
@jagonzalr See my comment below :) If it makes sense to you, then we don't need this change :)